app.py

```diff
@@ -12,13 +11,16 @@ def index():
 
 @app.route('/video')
 def video():
+    print "Video request"
+
     FNULL = open(os.devnull, 'w')
 
-    hri_cmd = ['python', 'heart_rate_deamon.py', video_file, opencv_path]
-    opencv_ps = subprocess.Popen(hri_cmd, stdout=subprocess.PIPE, stderr=FNULL)
+    client_proc_cmd = 'python heart_rate_client.py ' + video_file  
```

Comment:
I'm a bit confused as to what is the purpose of this? Why have python_heart_rate_clieant.py a subproccess where you could just have it as a function.